GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+
+### Stage 2: voting 
+When no consensus can be found, any core developer can call for a vote at any point during the discussion. 
+
+* The vote will conclude 10 working days from the call for the vote.
+* Votes are public and voluntary. Abstentions are allowed. You can abstain by simply not casting a vote. 
+* All votes are a binary vote: for or against accepting the proposed changes. 
+* Votes are casts as comments: +1 (approval) or -1 (rejection).
+
+For all types of changes, except appointments, the winning condition is a 2/3 majority of the votes casts by core developers including CC members. If the proposed change cannot gather a 2/3 majority of the votes cast by core developers (including CC members), the decision is escalated to the [Stage 3: conflict resolution](#Stage-3:-conflict-resolution). 
+
+For appointments, winning conditions vary depending on the role as described in the [section on roles](#Roles) above. Appointment decisions are not escalated to the CC. If the nomination cannot gather sufficient support, the nomination is rejected. 
+
+### Stage 3: conflict resolution
+If the proposed change cannot gather a 2/3 majority of the votes cast, the CC tries to resolve the deadlock. 
+
+* Any CC decision must be backed by an [enhancement proposal](#sktime-enhancement-proposals).
+* The CC will use consensus seeking.
+* If no consensus can be found within a month, the decision is made through a simple majority vote (with tie breaking) among the CC members. 
+
+### sktime enhancement proposals (STEPs)
+For all decision of the CC, an enhancement proposal must have been made public and discussed before the vote. 
+
+An enhancement proposal must be a consolidated document, with a concise problem statement, a clear description of the proposed solution and a comparison with alternative solutions, as outlined in our [template](https://github.com/sktime/enhancement-proposals/blob/master/TEMPLATE.md). 
+
+We collect and discuss proposals in sktime's [repository for enhancement-proposals](https://github.com/sktime/enhancement-proposals). 
+
+### Algorithm inclusion criteria
+
+Curation is about how we select contributions, which criteria we use in order to decide which contributions to include, and in which cases we deprecate and remove contributions.
+
+We have the following guidelines:
+
+* We only consider published algorithms which have been shown to be competitive in comparative benchmarking studies or practically useful in applied projects. A technique that provides a clear-cut improvement (e.g. an enhanced data structure or a more efficient approximation technique) on a widely-used method will also be considered for inclusion.
+* From the algorithms or techniques that meet the above criteria, only those which fit well within the current framework and API of sktime are accepted. For extending current frameworks and API, see the [process for major changes]().
```

Comment:
perhaps we should say that this is not a major obstacle, but it can happen that the framework needs to be extended *first* (technical conditionality)

Reply:
Oh there's also a link missing (empty parenthesis)